### DISCUSSION TITLE
TVDB for TV-Shows instead of TMDB

### DISCUSSION THREAD

**Discussion by joie96** [seerr-team/seerr#371] [Ideas]
Hello folks,

maby it's possible to use tvdb as metadata scraper for tv-shows, because sonarr only supports tvdb. If I monitor a tv-show in sonarr which has a different sorting in tvdb in comparision with tmdb, jellyseer will not notice.

This would be great.

Greetings

**Comment by fallenbagel** [comment]
Tvdb has paid api. So sorry this cannot be done

**Reply by VampiricAlien** [reply]
Could this in included for those who already have an API?
